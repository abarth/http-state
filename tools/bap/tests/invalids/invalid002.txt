; parsing indented rules with different indentation
  ABNF = test

    test = "test"
