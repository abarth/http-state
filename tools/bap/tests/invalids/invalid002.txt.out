invalids/invalid002.txt(3:8): error: Indented rules are Right Out.
; test UNDEFINED
ABNF = test
parsing failed: 1 errors encountered
